### DISCUSSION TITLE
Error during build

### DISCUSSION THREAD

**Discussion by KhomenkoSerhii** [rainbow-me/rainbowkit#2143] [Q&A]
Hi! In my project iam use: 
- react + vite
- @rainbow-me/rainbowkit": "^2.1.4",
-  "viem": "^2.19.4",
-    "wagmi": "^2.12.5",
- "@tanstack/react-query": "^5.51.23",

recently I started getting an error when making a build - error during build: [commonjs--resolver] node_modules/@rainbow-me/rainbowkit/dist/wallets/walletConnectors/chunk-JVYICMGM.js (27:67): Cannot use a reserved word as a shorthand property 25:     name: "TokenPocket",
26:     rdns: "pro.tokenpocket",
27:     iconUrl: async () => (await import("./tokenPocketWallet-BVMBOYTC.js")).default,
                                                                       ^
28:     iconBackground: "#2980FE",
29:     installed: !shouldUseWalletConnect ? isTokenPocketInjected : void 0,

RollupError: Cannot use a reserved word as a shorthand property ...

Delete node modules, re-install doesn't help.
 Should I configure the project  or how to fix this error?

**Comment by zebriot** [comment]
same issue.
